api/app/__init__.py

```diff
@@ -28,7 +30,9 @@ def create_app(test_config=None):
         # default secret that should be overriden in invorn or config
         SECRET_KEY=os.environ.get("SECRET_KEY", "dev"),
         SQLALCHEMY_DATABASE_URI=db_url,
-        SQLALCHEMY_TRACK_MODIFICATIONS=False
+        SQLALCHEMY_TRACK_MODIFICATIONS=False,
+        CORS_ALLOW_HEADERS=['Content-Type', 'Authorization'],
+        CORS_SUPPORTS_CREDENTIALS=True
```

Comment:
Here I had to add Cors in order to get the React App to communicate with the API